spec/index.html

```diff
@@ -351,6 +352,64 @@ <h3>Blank nodes</h3>
     </figure>
   </section>
 
+  <section id="section-triple-terms">
+    <h3>Triple terms</h3>
+
+    <p>As the basic unit of description in RDF, the triple is a simple
+    abstraction for stating a directed relationship between two resources.
+    Sometimes, we need to describe more detailed circumstances underlying such
+    relationships. While we could describe these using more sophisticated
+    modelling, there is a tension between simple, direct statements and more
+    complex, granular descriptions. This may require a trade-off between
+    directness and completeness.</p>
+
+    <p>To bridge this difference, we can use <a
+    data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a> as
+    <em>references</em> to simple statements. A triple term is a fixed object
```

Comment:
I am not sure what "fixed" is supposed to mean here. Are there any objects in an RDF graph, that are not "fixed"?

Reply:
No, I guess there aren't. (Except _maybe_ bnodes?)

That phrasing is an attempt to hint at the unicity ("type nature") of the propositions denoted by triple terms. But I'll seek to rephrase this as I'm working your other suggestions into the text. (Reordering; adding a figure; cutting or shortening.)